src/libcollections/string.rs

```diff
@@ -964,7 +961,7 @@ pub fn as_string<'a>(x: &'a str) -> DerefString<'a> {
     DerefString { x: as_vec(x.as_bytes()) }
 }
 
-#[unstable(feature = "collections", reason = "associated error type may change")]
+#[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
Right, but this might be one reason to leave it opaque for the time being -- so that if we get `Void` (which has come up from time to time, and is a tiny API) we could switch over to it.